src/webextension/list/manage/components/upgrade-account.js

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import { Localized } from "fluent-react";
+import PropTypes from "prop-types";
+import React from "react";
+import { connect } from "react-redux";
```

Comment:
Similar to the `AccountStatus` component, this should be split into two files with a presentational component in `components/` and a Redux-connected component in `containers/`. I'd also probably put the implementations of `doCreate` and `doSignIn` in the Redux-ified version.

More generally, perhaps we should pipe our `sendMessage` stuff through Redux too? I'll have to think about that some more...